sys/include/net/gnrc/ndp/internal.h

```diff
@@ -80,9 +80,12 @@ void gnrc_ndp_internal_send_nbr_sol(kernel_pid_t iface, ipv6_addr_t *tgt,
  *                          not be NULL.
  * @param[in] supply_tl2a   Add target link-layer address option to neighbor
  *                          advertisement if link-layer has addresses.
+ * @param[in] ext_opts      External options for the neighbor advertisement. Leave NULL for none.
```

Comment:
for the record: will `ext_ops` be `NULL` if it gets released due to an error? If not, I would also want you to add a return value to `gnrc_ndp_internal_send_nbr_adv`